symplyphysics/laws/gravity/escape_velocity.py

```diff
@@ -18,6 +24,31 @@
 
 law = Eq(velocity, sqrt(gravitational_constant * planet_mass / (radius + height)))
 
+# This law might be derived via "gravity_force_from_mass_and_distance" law, "acceleration_from_force" law,
+# "centripetal_acceleration_is_squared_velocity_by_radius" law.
+
+# The radius of the orbit consists of the radius of the planet and the height above its surface.
+centripetal_law_applied = centripetal_law.law.subs({
+    centripetal_law.linear_velocity: velocity,
+    centripetal_law.curve_radius: radius + height,
+})
+acceleration_derived = solve(centripetal_law_applied, centripetal_law.centripetal_acceleration, dict=True)[0][centripetal_law.centripetal_acceleration]
+
+acceleration_law_applied = acceleration_law.law.subs({
+    acceleration_law.acceleration: acceleration_derived,
+})
+force_derived = solve(acceleration_law_applied, acceleration_law.force, dict=True)[0][acceleration_law.force]
+
+gravity_force_law_applied = gravity_force_law.law.subs({
+    gravity_force_law.first_object_mass: planet_mass,
+    gravity_force_law.gravitational_force: force_derived,
```

Comment:
Add comment why gravitational force is equal to the force from centripetal acceleration.